kratos/python/add_strategies_to_python.cpp

```diff
@@ -519,6 +549,34 @@ namespace Kratos
                     .def("Info", &BaseSolvingStrategyType::Info)
                     ;
 
+            typedef ExplicitSolvingStrategy< SparseSpaceType, LocalSpaceType > BaseExplicitSolvingStrategyType;
+            py::class_<BaseExplicitSolvingStrategyType, typename BaseExplicitSolvingStrategyType::Pointer>(m, "ExplicitSolvingStrategy")
+                .def(py::init<ModelPart &, bool, int>())
+                .def(py::init<ModelPart&, typename ExplicitBuilderType::Pointer, bool, int>())
+                .def("Predict", &BaseExplicitSolvingStrategyType::Predict)
+                .def("Initialize", &BaseExplicitSolvingStrategyType::Initialize)
+                .def("IsConverged", &BaseExplicitSolvingStrategyType::IsConverged)
+                .def("SetEchoLevel", &BaseExplicitSolvingStrategyType::SetEchoLevel)
+                .def("GetEchoLevel", &BaseExplicitSolvingStrategyType::GetEchoLevel)
+                .def("SetRebuildLevel", &BaseExplicitSolvingStrategyType::SetRebuildLevel)
+                .def("GetRebuildLevel", &BaseExplicitSolvingStrategyType::GetRebuildLevel)
+                .def("SetMoveMeshFlag", &BaseExplicitSolvingStrategyType::SetMoveMeshFlag)
+                .def("MoveMeshFlag", &BaseExplicitSolvingStrategyType::MoveMeshFlag)
+                .def("MoveMesh", &BaseExplicitSolvingStrategyType::MoveMesh)
+                .def("Clear", &BaseExplicitSolvingStrategyType::Clear)
+                .def("Check", &BaseExplicitSolvingStrategyType::Check)
+                .def("InitializeSolutionStep", &BaseExplicitSolvingStrategyType::InitializeSolutionStep)
+                .def("FinalizeSolutionStep", &BaseExplicitSolvingStrategyType::FinalizeSolutionStep)
+                .def("SolveSolutionStep", &BaseExplicitSolvingStrategyType::SolveSolutionStep)
+                .def("GetModelPart", [](const BaseExplicitSolvingStrategyType& self) -> const ModelPart& { return self.GetModelPart(); })
```

Comment:
```suggestion
                .def("GetModelPart", [](BaseExplicitSolvingStrategyType& self) -> ModelPart& { return self.GetModelPart(); })
```

I am pretty convinced this should not be the const version, I think python anyway doesnt care abt const